### PR TITLE
Environments

### PR DESCRIPTION
**Pull request by elias-garcia**

### CODE REVIEW

File: src/environment.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import * as constants from './constants'
+import * as coordinatorApi from './api'
+import { ContractNames } from './constants'
```
Comment (AlbertoElias) [inline src/environment.js:3]: you're already importing this as constants

Comment (elias-garcia) [reply src/environment.js]: fixed!

File: src/environment.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    setContractAddress(constants.ContractNames.Hermez, PUBLIC_CONTRACT_ADDRESSES[env][constants.ContractNames.Hermez])
+    setContractAddress(constants.ContractNames.WithdrawalDelayer, PUBLIC_CONTRACT_ADDRESSES[env][constants.ContractNames.WithdrawalDelayer])
+    setBaseApiUrl(PUBLIC_BASE_API_URLS[env])
+    setBatchExplorerUrl(BATCH_EXPLORER_URLS[env])
+  }
+
+  if (typeof env === 'object') {
+    if (
+      env.contractAddresses &&
```
Comment (AlbertoElias) [inline src/environment.js:68]: maybe wrap both in an if env.contractAddresses? I think it would be easier to read

**Review by AlbertoElias — APPROVED** [review]
Small change suggestion

File: src/environment.js (hunk 2 above)
Comment (elias-garcia) [reply src/environment.js]: agree! done

**Pull request merged**